css/about.css

```
@@ -80,13 +80,13 @@ p {
 /* ========================== Variables ========================== */
 
 :root {
-
+    --font-size: .8rem;
```

is something when wrong here
```suggestion
    font-size: .8rem;
```